doc/Installation/Installation-CentOS-7-Apache.md

```diff
@@ -0,0 +1,132 @@
+n> NOTE: These instructions assume you are the root user.  If you are not, prepend `sudo` to the shell commands (the ones that aren't at `mysql>` prompts) or temporarily become a user with root privileges with `sudo -s` or `sudo -i`.
```

Comment:
Perhaps a not regarding selinux is also needed(If selinux is activated pr. default you won't be allowed to create httpd log files for example)